b2sdk/account_info/abstract.py

```diff
@@ -10,6 +10,7 @@
 
 from abc import abstractmethod
 
+from . import exception
```

Comment:
do not use relative imports and absolute imports together. The style of this project is to use absolute imports, so use that.